src/libos/src/util/resolv_conf_util.rs

```diff
@@ -0,0 +1,21 @@
+use super::*;
+use crate::fs::{AccessMode, CreationFlags, FsView};
+
+pub fn write_resolv_conf() -> Result<()> {
+    const RESOLV_CONF_PATH: &'static str = "/etc/resolv.conf";
+    let fs_view = FsView::new();
+    // overwrite /etc/resolv.conf if existed
+    let resolv_conf_file = fs_view.open_file(
+        RESOLV_CONF_PATH,
+        AccessMode::O_RDWR as u32 | CreationFlags::O_CREAT.bits() | CreationFlags::O_TRUNC.bits(),
+        0o666,
+    )?;
+    let resolv_conf_str = RESOLV_CONF_STR.read().unwrap().clone();
```

Comment:
Since no other thread expects to get the lock, does no need "clone".

Reply:
Done